functions/supplier/zeromq-supplier/src/test/java/org/springframework/cloud/fn/supplier/zeromq/ZeroMqSupplierConfigurationTests.java

```diff
@@ -39,37 +42,41 @@
  * @author Daniel Frey
  * since 3.1.0
  */
+@SpringBootTest(properties = {"zeromq.supplier.topics=test-topic"})
```

Comment:
Add `@DirtiesContext`, please, so our application context is destroyed properly.